dandischema/models.py

```diff
@@ -820,26 +869,36 @@ class CommonModel(DandiBaseModel):
         nskey="schema",
     )
     studyTarget: Optional[List[str]] = Field(
-        None, description="What the study is related to", nskey="dandi"
+        None,
+        description="What the objectives or specific questions of the study are.",
+        nskey="dandi",
     )
     license: Optional[List[LicenseType]] = Field(
-        None, description="Licenses associated with the item.", nskey="schema"
+        None,
+        description="Licenses associated with the item. DANDI only supports a "
+        "subset of Creative Commons Licenses (creativecommons.org) "
+        "relevant to datasets.",
```

Comment:
The issue that kicked off this ticket was that licenses have abbreviations that I do not understand and I doubt other users would either. Could you spell out what these options are and how they are different from each other?

Reply:
we may keep changing the list and i did not want to describe all of them here. simply a place to go to, to look. providing clear guidance on choices would be in the domain of the handbook or some tutorial, not this descriptor i think.